### PR TITLE
Change plugin to one that supports vc-jose-cose with VCDM 2.0

### PR DESCRIPTION
**Pull request by andresuribe87**
We had multiple examples of "Verifiable Credentials" that weren't conforming to VCDM 2.0. The previous respec plugin that we were using had the `vc` claim inside the JWT payload, which was VCDM 1.1 JWTs. 

This change makes it so all examples are VCDM 2.0 JWTs secured according to vc-jose-cose. It makes it easier for readers by having explicit examples of VCs that are secured.

In the future, we may want to add internally secured VCs as part of the examples.

***

### :boom: Error: 500 Internal Server Error :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on Aug 9, 2023, 7:39 PM UTC)_.

<details>
<summary>More</summary>

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [Spec Generator](https://www.w3.org/2015/labs/) - Spec Generator is the web service used to build specs that rely on ReSpec.

:link: [Related URL](https://labs.w3.org/spec-generator/?type=respec&url=https%3A%2F%2Fraw.githubusercontent.com%2Fw3c%2Fvc-json-schema%2Fab633fed4c831adc4bef4baa4c018034da3ef0e3%2Findex.html%3FisPreview%3Dtrue)

```

😭  Sorry, there was an error generating the HTML. Please report this issue!
Specification: http://labs.w3.org/spec-generator/uploads/7Baf0k/index.html?isPreview=true%3FisPreview%3Dtrue&publishDate=2023-08-09
ReSpec version: 34.1.6
File a bug: https://github.com/w3c/respec/
Error: undefined

```

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20w3c/vc-json-schema%23196.)._
</details>

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

**Comment by decentralgabe** [comment]
@andresuribe87 could you also replace instances of `VC-JWT` with `VC-JOSE-COSE`?

**Comment by TallTed** [comment]
There are a number of other occurrences of `vc-jwt` that should probably also become `vc-jose-cose`. I'm not certain of these, so I'm not putting in suggestions. Hopefully someone else more certain can review...

**Comment by decentralgabe** [comment]
the only other I see is https://github.com/w3c/vc-json-schema/blob/main/index.html#L524

File: index.html
Comment (TallTed) [inline index.html:12]: `vc-jwt` -> `vc-jose-cose` ?

File: index.html
Comment (TallTed) [inline index.html:206]: `vc-jwt` -> `vc-jose-cose` ?

File: index.html
Comment (TallTed) [inline index.html:309]: `vc-jwt` -> `vc-jose-cose` ?

File: index.html
Comment (TallTed) [inline index.html:336]: `vc-jwt` -> `vc-jose-cose` ?

File: index.html
Comment (TallTed) [inline index.html:536]: `vc-jwt` -> `vc-jose-cose` ?

File: index.html
Comment (TallTed) [inline index.html:599]: `vc-jwt` -> `vc-jose-cose` ?

File: index.html
Comment (TallTed) [inline index.html:622]: `vc-jwt` -> `vc-jose-cose` ?

File: index.html
Comment (TallTed) [inline index.html:668]: `vc-jwt` -> `vc-jose-cose` ?

File: index.html
Comment (TallTed) [inline index.html:890]: `vc-jwt` -> `vc-jose-cose` ?

File: index.html
Comment (TallTed) [inline index.html:938]: `vc-jwt` -> `vc-jose-cose` ?

**Comment by andresuribe87** [comment]
@TallTed all the occurrences of `vc-jwt` you highlighted are related to the implementation of the respec plugin we're using to render VCs in the spec. You can see the plugin's documentation at https://github.com/transmute-industries/respec-vc-jwt

In order to update these occurrences, we'd first need to:
* Update the plugin repo name. 
* Change how the plugin works so that it scans html nodes with the `.vc-jws` as part of the class attribute.
* Cut a new plugin release.

CC @OR13 for thoughts on the above.

I can create an issue to track that work, but I don't think it should block this PR.

**Comment by OR13** [comment]
I'd prefer not to update the plugin until the spec updates land... Meaning both the open PRs in vc-jose-cose are merged. I'm fine updating the plugin after that... It's also ok to not use the plugin and just hand craft examples

File: index.html
Comment (OR13) [reply index.html:938]: the plugin won't understand this... If `vc-jose-cose` can make progress, I will rewrite the plugin to better handle the changes since it was originally written.

File: index.html
Comment (OR13) [reply index.html:12]: I'm going to resolve these comments, the plugin was built before the rename, and it won't be updated until after the work item makes progress on media types.

**Comment by OR13** [comment]
@TallTed  I resolved a lot of your comments, without addressing them, because they were all duplicated of https://github.com/w3c/vc-json-schema/pull/196#discussion_r1291324792

**Comment by TallTed** [comment]
> [@OR13] @TallTed I resolved a lot of your comments, without addressing them, because they were all duplicated of https://github.com/w3c/vc-json-schema/pull/196#discussion_r1291324792

That's fine. I wasn't sure whether they should change; hence the `?` that each ended in.

**Pull request merged**
